network/iptables.go

```diff
@@ -130,11 +182,21 @@ func DeleteIPTables(rules []IPTablesRule) error {
 		log.Errorf("Failed to setup IPTables. iptables binary was not found: %v", err)
 		return err
 	}
-	teardownIPTables(ipt, rules)
+	iptRestore, err := NewIPTablesRestore()
+	if err != nil {
+		// if we can't find iptables, give up and return
+		log.Errorf("Failed to setup IPTables-restore: %v", err)
```

Comment:
nit: this should probably be `iptables-restore` instead of `IPTables-restore`

Reply:
:ok_hand: